packages/search-in-workspace/src/browser/search-in-workspace-frontend-contribution.ts

```diff
@@ -55,12 +57,13 @@ export class SearchInWorkspaceFrontendContribution extends AbstractViewContribut
     }
 
     async initializeLayout(app: FrontendApplication): Promise<void> {
-        await this.openView({activate: false});
+        await this.openView({ activate: false });
     }
 
     registerCommands(commands: CommandRegistry): void {
         super.registerCommands(commands);
         commands.registerCommand(SearchInWorkspaceCommands.OPEN_SIW_WIDGET, {
+            isEnabled: () => this.workspaceService.tryGetRoots().length > 0,
```

Comment:
When entering data in the search field, you activate the "Refresh" button which is not necessary since there is no workspace to search !

Reply:
I tried to have as minimal impact as possible to the code so I reuse the same notification and only display the message if the user in fact attempts to perform a search when no workspace is present, I don't persist the message or disable any of the buttons

Reply:
Anyways @lmcbout, this isn't the location of where I add the checks, this only disables the menu item, as well as the quick-open-command for when no workspace is present and a user attempts to search.

Reply:
I addressed the issue of not enabling the `Refresh` button if no workspace is present. 
This snippet however is not the cause of that issue. This `enable` only controls whether the `menu` and `command` for `Close Workspace` is enabled or not.